.flake8

```diff
@@ -70,6 +70,7 @@ per-file-ignores =
   test/TestBaseFormatter.py: D100 D103
   test/TestBecomeUserWithoutBecome.py: PT009 D100 D101 D102
   test/TestCliRolePaths.py: PT009 D100 D101 D102
+  test/TestCommandsInsteadOfModulesRule.py: PT009
```

Comment:
Please avoid adding new exceptions, just use normal asserts to avoid it. Our desire was to remove the others but we always had bigger fixed to make.

Reply:
Will fix.